resync/scripts/install_redis.py

```
@@ -0,0 +1,350 @@
+      test: ["CMD", "redis-cli", "ping"]
+      interval: 10s
+      timeout: 3s
+      retries: 3
+
+volumes:
+  redis_data:
+    driver: local
+"""
+
+
+def run_command(cmd: list[str], check: bool = True, cwd: str = None) -> subprocess.CompletedProcess:
```

Ruff (0.15.0)

[error] Boolean-typed positional argument in function definition

(FBT001)